ddapm_test_agent/trace_snapshot.py

```diff
@@ -319,6 +405,15 @@ def _snapshot_trace_str(trace: Trace, removed: Optional[List[str]] = None) -> st
                     span["meta"].pop(key[5:], None)
                 elif key.startswith("metrics."):
                     span["metrics"].pop(key[8:], None)
+                elif key.startswith("span_links.attributes."):
+                    if "span_links" in span:
+                        for link in span["span_links"]:
+                            if "attributes" in link:
+                                link["attributes"].pop(key[22:], None)
```

Comment:
Oh, I see that we remove that added data here.

I still wonder if it would be better to store the attribute data separate from the span data itself..

Reply:
Why? It's ultimately part of the span.

Reply:
Yep I misread above